tests/robot-tests/tests/admin_and_public/bau/data_reordering.robot

```diff
@@ -530,9 +530,7 @@ Validate table cells
     user checks results table cell contains    10    2    39,761,905
 
 Go back to locations step
-    # TODO not sure why button with test Id isn't found? Using label instead
-    # user clicks button    testid:wizardStep-3-goToButton
-    user clicks button    Edit locations
+    user clicks element    testid:wizardStep-3-goToButton
```

Comment:
Just a personal preference, but I actually prefer the label version of this, because it's more obvious to a reader of the test as to what "wizardStep-3-goToButton" is actually doing.